apps/files/src/components/UploadProgress.vue

```diff
@@ -9,25 +9,30 @@
         id="files-upload-progress-dropdown-trigger"
         icon="expand_more"
       />
-      <translate
-        :translate-n="count"
-        translate-plural="Uploading %{ count } items"
-      >
-        Uploading %{ count } item
-      </translate>
+      <span id="upload-label">
+        <translate
```

Comment:
Translate tag is a span in default. You can assign the id directly and avoid one extra tag.

Reply:
Thx. Fixed.